pkg/kv/kvserver/replica_raftstorage.go

```diff
@@ -911,8 +911,6 @@ func clearSubsumedReplicaDiskData(
 		subsumedReplSST := storage.MakeIngestionSSTWriter(
 			ctx, st, subsumedReplSSTFile,
 		)
-		//nolint:deferloop TODO(#137605)
-		defer subsumedReplSST.Close()
```

Comment:
because it's fine to leak these, they'll just get garbage collected?

Reply:
We already either `Close()` or `Finish()` in all code paths below (and `Close()` is a no-op after `Finish()`).